### ISSUE TITLE
Error in cr_build_make -- unused argument (projectId = projectId)

### ISSUE THREAD

**Issue by anirban-mukherjee** [MarkEdmondson1234/googleCloudRunner#57]
When I call cr_deploy_plumber, I get:

2020-04-20 21:58:00> Compressing files from /deploy to api.tar.gz
2020-04-20 21:58:00> Uploading api.tar.gz to #######/api.tar.gz
2020-04-20 21:58:00 -- File size detected as 781 bytes
Error in cr_build_make(yaml = x, source = source, timeout = timeout, images = images,  : 
  unused argument (projectId = projectId)

where ####### is my project name.

I get the same error with both the example file in googleCloudRunner and my own file. As cr_build_make does not seem to require projectId, it looks like there may be a bug somewhere in that either cr_build_make should require projectId or should need to be passed projectId.

**Comment by anirban-mukherjee** [comment]
Even when installing the library, I get a warning that there is a possible error in cr_build_make.

* installing *source* package ‘googleCloudRunner’ ...
** using staged installation
** R
** inst
** byte-compile and prepare package for lazy loading
Note: possible error in 'cr_build_make(yaml = x, ': unused argument (projectId = projectId) 
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (googleCloudRunner)

**Comment by MarkEdmondson1234** [comment]
I think you are using the dev version on GitHub and caught a commit where this occurred, so you can try installing the latest (as I’m in the middle of fixing) or if trying it out first time probably best to install the more stable version from CRAN

**Comment by anirban-mukherjee** [comment]
Thanks. Confirming that this issue is fixed in the development version. I have reverted to the CRAN version since you are actively developing the codebase and there are likely to be other unresolved issues. Thanks for all your work on this and the other great packages.

**Comment by MarkEdmondson1234** [comment]
Thanks! Look out for a new CRAN release soon

________________________________

**Comment by anirban-mukherjee** [comment]
Two smaller issues to report for the document at https://code.markedmondson.me/googleCloudRunner/articles/cloudrun.html

1. myMessage in the cr_dockerfile_plumber does not seem to be available in the namespace. I had to comment it out.

2. The system.file command is passed cloudRunner and not googleCloudRunner as the package name. I changed that to googleCloudRunner.

Else the rest worked like a charm. Thanks again!

Anirban
